doc/ref_model/chapters/appendix-a.md

```diff
@@ -16,19 +16,75 @@ This Appendix has two goals:
+
+-	Encapsulation = Restricting of direct access to some of an object's components. [2]
+
+-	Appliance deployment model = Application has tight coupling with underlying Platform even if the application is virtualized or containerized.
+
+-	Tenant/Cloud deployment model = Tenant brings applications that are decoupled from the platform provided by Cloud operator. Tenant and Cloud operator are normally different legal entities.
+
+-	Application Control = Any way of controlling tenant’s application. Depending on RA and technologies used, it can be tenant’s VNFM or Orchestration SW, or Platform capability.
+
+-	Decomposition = Decomposition (also known as factoring) is breaking a complex system into parts that are easier to program and maintain. [3]
+
+-	Resilience = Resilience is the ability to provide and maintain an acceptable level of service in the face of various faults and challenges to normal operation. [4]
+
+-	Observability = Observability is a measure of how well internal states of a system can be inferred from knowledge of its external outputs. [5]
+
 
 <a name="A.3"></a>
 ## A.3 VNF Maturity Levels
 (And how they map to CNTT RAs)
 
+Taking advantage of CNTT pre-defined environments with common capabilities, applications can be developed and deployed more rapidly, providing more service agility and easier operations. The extent to which this can be achieved will depend on levels of decoupling between application and infrastructure or platform underneath the application:
+
+**1. Infrastructure**:
+- a. Application functionality or application control require infrastructure components beyond RM profiles or requires infrastructure configuration changes beyond RA exposed APIs. Generally, such application is tightly coupled with infrastructure which results in Appliance deployment model. 
+- b. Application control using RA APIs finds node with required infrastructure component(s), and in that node using RA APIs configures infrastructure components that make application work. Example is application that requires certain acceleration adapter available in RM profile and is exposed through RA APIs.
+- c. Application control using RA APIs finds node with optional infrastructure component(s), and in that node using RA APIs configures infrastructure component(s) that make application work better (like more performant) than without that infrastructure component. Example is app that would work better with certain acceleration adapter but can also work without it.
+- d. Application control using RA APIs finds general profile node without any specific infrastructure component.
+
+**2. Platform as a Service** (as per RA2 WIP: common services starting from Load Balancer and Store State, also typically includes Logging and Telemetry):
```

Comment:
the comment in parentheses should be removed as RA1 WIP also includs logging etc. while RA1 existing content addresses load balancers etc.

Reply:
Done in #1376 .